cdc/kv/client.go

```diff
@@ -1323,11 +1328,17 @@ func (s *eventFeedSession) sendResolvedTs(
 	worker *regionWorker,
 	addr string,
 ) error {
-	statefulEvents := make([][]*regionStatefulEvent, worker.inputSlots)
+	statefulEvents := make([]*regionStatefulEvent, worker.inputSlots)
```

Comment:
It's better to use value type instead of pointer type to reduce memory allocation. Because we can determine the slice length.

Reply:
ok, I will do it in another PR if needed.